ecr/fetcher_test.go

```diff
@@ -199,7 +200,7 @@ func TestFetchManifestNotFound(t *testing.T) {
 func TestFetchLayer(t *testing.T) {
 	registry := "registry"
 	repository := "repository"
-	layerDigest := "digest"
+	layerDigest := testdata.LayerDigest.String()
```

Comment:
Since we're not enforcing a valid digest (you removed that validation at my request), I think we can also drop this part of the change as well.  I like to pick test data that is either self-explanatory (like "digest" being the digest) or fulfilling a specific purpose.  We no longer require a real digest here, and a future author looking at the code might wonder why we're going to the extra effort of ensuring that we have one.

Reply:
I've updated the latest changes to use a descriptive string in place of a valid digest. The names of the provided consts should be self-describing in what their intended use is and are entirely bogus values.